### PR TITLE
[common-artifacts] Enable Gather test

### PR DESCRIPTION
**Pull request by seanshpark**
This will enable Gather Op test.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #8241, from draft #8245

**Comment by seanshpark** [comment]
@mhs4670go , PTAL

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/common-artifacts/options.lst [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 #[[ tcgenerate_option : add additional option(s) for generation ]]
 
-# TODO add options
+# make valid 'indices' input value
+tcgenerate_option(Gather_001 --input_range indices 0 3)
```
Comment (mhs4670go) [inline compiler/common-artifacts/options.lst:6]: `Gather_000` isn't needed?

Comment (seanshpark) [reply compiler/common-artifacts/options.lst:6]: 000 has constant incices with explicit values.
```
operand {
  name: "indices"
  type: INT32
  shape { dim: 2 }
  filler { tag: "explicit" arg: "1" arg: "2" }
}
```

**Pull request merged**